src/index.js

```diff
@@ -533,17 +533,45 @@ export default class RNPickerSelect extends PureComponent {
         );
     }
 
+    renderWeb() {
+        const { disabled, Icon, style, pickerProps } = this.props;
+        const { selectedItem } = this.state;
+
+        return (
+            <View style={[defaultStyles.viewContainer, style.viewContainer]}>
+                <Picker
+                    style={[
+                        Icon ? { backgroundColor: 'transparent' } : {}, // to hide native icon
+                        style.inputWeb,
+                        this.getPlaceholderStyle(),
+                    ]}
+                    testID="web_picker"
+                    enabled={!disabled}
+                    onValueChange={this.onValueChange}
+                    selectedValue={selectedItem.value}
+                    {...pickerProps}
+                >
+                    {this.renderPickerItems()}
+                </Picker>
+                {this.renderIcon()}
```

Comment:
does this work on web?

Reply:
The icon or the picker item?

Reply:
Yes, icons work on the web

Reply:
We may want to change the style for how they're displayed - it doesn't look amazing out of the box, but it does work:

<img width="524" alt="Screen Shot 2020-04-23 at 8 14 35 PM" src="https://user-images.githubusercontent.com/10475533/80161878-be773880-859f-11ea-9175-d812e135fe50.png">

Reply:
is there an easy way to disable the native one underneath? if so, i think it's worth the effort to tweak this base style - if not, maybe we should just kill off the custom icon support for this first pass.

Reply:
There is an easy way to do this, yes, you just pass in:

```
WebkitAppearance: "none",
MozAppearance: "none",
appearance: "none",
```

To the inputWeb object (or whatever we decide on for web specific styles)

<img width="1011" alt="Screen Shot 2020-04-29 at 1 13 48 PM" src="https://user-images.githubusercontent.com/10475533/80625493-4d120c80-8a1b-11ea-8b79-dc0fb8389525.png">

We'd need to figure out how we want to modify the styles for the icon

Reply:
In the image above, I removed the default style, and replaced it with a chevron down from Font Awesome.

Reply:
alright so sounds like the answers are:

1. keep it a separate method
2. kill `Icon ? { backgroundColor: 'transparent' } : {},` line
3. kill `this.getPlaceholderStyle(),` line

other than that - just needs tests

Reply:
Ok, I'll clean it up and write tests sometime in the next week or two